addons/udes_stock/data/routes.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+
+    <!-- Create "Goods In, Putaway" route -->
+    <record id="route_in" model="stock.location.route">
+      <field name="name">Inbound</field>
+      <field name="sequence" eval="10"/>
+      <field name="warehouse_selectable" eval="True"/>
+      <field name="warehouse_ids" eval="[(6, False, [ref('stock.warehouse0')])]"/>
```

Comment:
How about adding a warehouse not resetting it to only stock.warehouse0, maybe can be done with 4, ref('stock.warehouse0') not sure about the proper syntax. 
I am saying this just in case that if in future we want to apply multi company and we may add another warehouse on fly for the same route. 
I have a concern in general with data that we dont use noupdate=1 even if we use we ran the service as -i which it will think that is a new instalment and will override values changed from users.
Maybe thinking to much :)

Reply:
I don't believe there are any plans for multi-warehouse systems in the future